--- a/vendor/roundcube/plugin-installer/src/Roundcube/Composer/ExtensionInstaller.php	2022-01-30 12:41:58.000000000 +0100
+++ b/vendor/roundcube/plugin-installer/src/Roundcube/Composer/ExtensionInstaller.php.new	2022-01-31 22:26:20.000000000 +0100
@@ -80,14 +80,23 @@ class ExtensionInstaller extends Library
             // initialize database schema
             if (!empty($extra['roundcube']['sql-dir'])) {
                 if ($sqldir = realpath($package_dir . DIRECTORY_SEPARATOR . $extra['roundcube']['sql-dir'])) {
-                    $self->io->write("<info>Running database initialization script for $package_name</info>");
-
-                    $roundcube_version = self::versionNormalize(RCMAIL_VERSION);
-                    if (self::versionCompare($roundcube_version, '1.2.0', '>=')) {
-                        \rcmail_utils::db_init($sqldir);
+                    if (getenv('SKIP_DB_INIT')) {
+                        $self->io->write("<warn>Skipping database initialization script for $package_name</warn>");
+                        $self->io->write("<warn>Please run on container init: \\rcmail_utils::db_init('$sqldir');</warn>");
+                        // add directory to .plugin_db_init for container start
+                        $fp = fopen('/var/www/.db_init', 'a');
+                        fwrite($fp, $package_name . ":" . $sqldir . "\n");
+                        fclose($fp);
                     }
                     else {
-                        throw new \Exception("Database initialization failed. Roundcube 1.2.0 or above required.");
+                        $self->io->write("<info>Running database initialization script for $package_name</info>");
+                        $roundcube_version = self::versionNormalize(RCMAIL_VERSION);
+                        if (self::versionCompare($roundcube_version, '1.2.0', '>=')) {
+                            \rcmail_utils::db_init($sqldir);
+                        }
+                        else {
+                            throw new \Exception("Database initialization failed. Roundcube 1.2.0 or above required.");
+                        }
                     }
                 }
             }
